Magic/Framework/Magic.cs

```diff
@@ -354,7 +354,7 @@ private static void OnWarped(object sender, WarpedEventArgs e)
                     I18n.Event_Wizard_9(),
                     I18n.Event_Wizard_Abovehead()
                 );
-                e.NewLocation.currentEvent = new Event(eventStr, MagicConstants.LearnedMagicEventId);
+                e.NewLocation.currentEvent = new Event(eventStr, "fromAssetName"/*todo*/, MagicConstants.LearnedMagicEventId);
```

Comment:
Note this todo. I'm too new with SV modding in general and space in particular to understand where this should be fetched from (if at all).